Agents/ForecastingAgent/forecasting/flaskapp/wsgi.py

```diff
@@ -0,0 +1,14 @@
+################################################
+# Authors: Magnus Mueller (mm2692@cam.ac.uk)   #
+# Date: 30 Nov 2022                            #
+################################################
+
```

Comment:
Add the following import settings here to avoid unnecessary logging output from py4jps. For details see: https://github.com/cambridge-cares/TheWorldAvatar/blob/main/Agents/MetOfficeAgent/agent/flaskapp/wsgi.py
```
# To avoid unnecessary logging information from py4j package, set logger level before 
# first creation of JPS_BASE_LIB module view (i.e. jpsBaseLibView = jpsBaseLibGW.createModuleView())
import logging
logging.getLogger("py4j").setLevel(logging.INFO)
```